ui/panels-plugin/src/plugins/time-series-chart/PanelComponent.stories.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright 2023 The Perses Authors
```

Comment:
I'd be down to call this file `TimeSeriesChartPanel.stories.tsx` (if I'm correctly understanding what we're making stories for)

Reply:
Mainly because that's more specific + searchable for me

Reply:
Or maybe TimeSeriesChartPanelComponent.stories.tsx is more correct?

Reply:
The names of all these will change once we fix exports in the future, having to import TimeSeriesChart.PanelComponent is kind of a hack, it'd be great if consumers could import from @perses-dev/panels-plugin like [Shan described here](https://github.com/perses/perses/pull/1032#discussion_r1153321529)

Reply:
I was trying to keep them shorter to avoid needing a massive left nav bar and that you get the time series bit from the folder, but the search use case is a good point. Lemme take a stab at giving these clearer names.

Reply:
> The names of all these will change once we fix exports in the future, having to import TimeSeriesChart.PanelComponent is kind of a hack, it'd be great if consumers could import from @perses-dev/panels-plugin like [Shan described here](https://github.com/perses/perses/pull/1032#discussion_r1153321529)

Totally agreed. Intentionally not trying to fix in this PR, which is why I have the note about not recommending importing from these components. It's fine if Shan does it for an experiment, but I wouldn't consider it a supported use case until we fix the exports.